### PR TITLE
feat(ratings) add component - INNO-884

### PR DESCRIPTION
**Pull request by raduchiriac**

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *ema-styleguide* ready!

Built with commit 648169363962f27bb0e5636e61983a054344c1ca

https://deploy-preview-16--ema-styleguide.netlify.com

File: framework/components/ema-ratings/ema-ratings.twig [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{#
+  Accepts:
+    - title (string): title of the panel (default: '')
+    - default_rating (string): set a default rating (default: 0)
+    - name (string): name of your form group input
+    - disabled (boolean): read-only rating (default: false)
+#}
+
+{# Internal properties #}
+{% set _css_class = 'ema-ratings' %}
```
Comment (emeryro) [inline framework/components/ema-ratings/ema-ratings.twig]: We usually use singular name for css classes (so `ema-rating`)

File: framework/components/ema-ratings/ema-ratings.twig [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  <div class="ema-ratings__stars">
+    {% for i in 5..1 %}
+      <input
+        type="radio"
+        class="ema-ratings__input"
+        aria-hidden="true"
+        id="ema-ratings__{{_name}}--{{i}}"
+        name={{ _name }}
+        tabindex={{ i-1 }}
+        value={{ i }}
```
Comment (emeryro) [inline framework/components/ema-ratings/ema-ratings.twig]: Missing quotes on the 3 previous values.

File: framework/templates/ecl-rating-form/ecl-rating-form.twig [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<form class="ma-rating-form">
+  <h3 class="ecl-heading ecl-heading--h3">How useful was this page?</h3>
+  <div class="ecl-u-d-flex">
+    {% for rating in ratings %}
+      <div class="ecl-u-mr-l">
+        {% include '@ec-europa/ema-ratings' with { rating } %}
```
Comment (emeryro) [inline framework/templates/ecl-rating-form/ecl-rating-form.twig]: For some reasons, the first rating is displayed twice, and not the "average" one.

File: framework/templates/ecl-rating-form/ecl-rating-form.twig [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      </div>
+    {% endfor %}
+    <a class="ecl-link" href="#">View all 12 ratings</a>
+  </div>
+  <div class="ecl-u-d-flex ecl-u-align-items-end ecl-u-mt-s">
+    {% include '@ec-europa/ecl-forms-textareas' with { rows: 3, extraClass: 'ecl-u-mr-s' } %}
+    {% include '@ec-europa/ecl-buttons' with {
```
Comment (emeryro) [inline framework/templates/ecl-rating-form/ecl-rating-form.twig]: To be updated once ema-buttons are merged

File: framework/components/ema-ratings/ema-ratings.twig (hunk 2 above)
Comment (raduchiriac) [reply framework/components/ema-ratings/ema-ratings.twig]: They are not mandatory

File: framework/templates/ecl-rating-form/ecl-rating-form.twig (hunk 3 above)
Comment (raduchiriac) [reply framework/templates/ecl-rating-form/ecl-rating-form.twig]: It's because `with { rating }` should be `with rating`

**Pull request merged**